ldi-nifi/ldi-nifi-processors/version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/config/VersionMaterialisationProcessorProperties.java

```diff
@@ -1,29 +1,40 @@
 package be.vlaanderen.informatievlaanderen.ldes.ldi.processors.config;
 
 import org.apache.nifi.components.PropertyDescriptor;
+import org.apache.nifi.processor.ProcessContext;
 import org.apache.nifi.processor.util.StandardValidators;
 
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+
 public class VersionMaterialisationProcessorProperties {
 
 	private VersionMaterialisationProcessorProperties() {
 	}
 
-	public static final PropertyDescriptor IS_VERSION_OF = new PropertyDescriptor.Builder().name("isVersionOf")
-			.displayName("Predicate used for isVersionOf")
+	public static final PropertyDescriptor RESTRICT_TO_MEMBERS = new PropertyDescriptor.Builder()
+			.name("RESTRICT_TO_MEMBERS")
+			.displayName("Restrict output to members")
+			.description("When enabled, only the member and the blank nodes references are included.")
+			.required(false)
+			.defaultValue(FALSE.toString())
+			.allowableValues(FALSE.toString(), TRUE.toString())
+			.addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+			.build();
+
+	public static final PropertyDescriptor VERSION_OF_PROPERTY = new PropertyDescriptor.Builder()
+			.name("VERSION_OF_PROPERTY")
```

Comment:
Add the same description and display name as the other properties for VERSION_OF_PROPERTY to make it consistent.